manifests/plugin.pp

```diff
@@ -48,6 +48,18 @@
 #   Default value: None
 #   This variable is optional
 #
+# [*proxy_username*]
+#   Proxy auth username to use when installing the plugin
+#   Value type is string
+#   Default value: None
+#   This variable is optional
+#
+# [*proxy_password*]
+#   Proxy auth username to use when installing the plugin
```

Comment:
Too fast copy/paste. I'm sure that youwanted to write password instead.
's/username/password/'